src/safeds/ml/classical/regression/_arima.py

```diff
@@ -0,0 +1,209 @@
+from __future__ import annotations
+
+import io
+import itertools
+
+import matplotlib.pyplot as plt
+import xxhash
+from statsmodels.tsa.arima.model import ARIMA
+
+from safeds.data.image.containers import Image
+from safeds.data.tabular.containers import Column, Table, TimeSeries
+from safeds.exceptions import (
+    DatasetMissesDataError,
+    MissingValuesColumnError,
+    ModelNotFittedError,
+    NonNumericColumnError,
+    NonTimeSeriesError,
+)
+
+
+class ArimaModelRegressor:
+    """Auto Regressive Integrated Moving Average Model."""
+
+    def __hash__(self) -> int:
+        """
+        Return a deterministic hash value for a regressor.
+
+        Returns
+        -------
+        hash : int
+            The hash value.
+        """
+        return xxhash.xxh3_64(
+            self.__class__.__qualname__.encode("utf-8")
+            + (1 if self.is_fitted() else 0).to_bytes(1)
+            + (bytes((9, 9, 9)) if self._order is None else bytes(self._order)),
```

Comment:
This only works if the integers in `_order` are between 0 and 256 (see https://docs.python.org/3/library/stdtypes.html#bytes-objects). If this is guaranteed to be always the case, this should be fine.

Otherwise, change it to something like this: (imports to `functools` and `operator` would be needed in this case)
```suggestion
            + (functools.reduce(operator.add, [i.to_bytes(8) for i in (9,9,9)], b'') if self._order is None else functools.reduce(operator.add, [i.to_bytes(8) for i in self._order], b'')),
```

Reply:
1. Right now it only takes integers from 0 to 2, I don't think it will go over 200.

But It's good to have a solution in mind, thank you